ui/src/telegrafs/components/TelegrafOutputOverlay.test.tsx

```diff
@@ -2,6 +2,11 @@
 import React from 'react'
 import {render} from 'react-testing-library'
 
+// NOTE: stubbing is required here as the CopyButton component
```

Comment:
I'd be curious to know why this was moved out?

Reply:
@hoorayimhelping moved it because it previously was failing. There was a strange error that arose when the underlying component used `DapperScrollbars` instead of `FancyScrollbars`